src/test/java/test/LoginTest.java

```diff
@@ -30,8 +33,9 @@ public void setup() throws InterruptedException{
     @Test
     public void loginTest() {
         homepage = new HomePage(driver);
-        homepage.clickOnSignInButton()
-                .setEmailAddressField("razvanratiu@yahoo.com")
+        loginPage = new LogInPage(driver);
```

Comment:
Since you opted to use the Fluent Page Object Pattern and the login page is the page that opens after clicking the Sign in button, then the loginPage object should be initialized after the action that triggers the page to open is called.
To better understand this concept:
- for each page class create a method which will wait for an element from the page (unique element) to open
- this method should be called each time a new object of a page class is created ([hint](https://www.javatpoint.com/java-constructor))